src/handler/dynamics.ts

```diff
@@ -4,7 +4,7 @@ import type {
   Context,
 } from 'aws-lambda';
 import { sendBooking } from '../services/eventbridge';
-import { IDynamicsBooking } from '../interfaces/IDynamicsBooking';
+import { IBooking } from '../interfaces/IBooking';
```

Comment:
adding `I` in front of interface names is not typescript practice but c#
Is this how interfaces being named across the typescript projects?

Reply:
As per style recommendation there should not be `I` prefix before interface names. This is recommended by both Microsoft and Google
If you add this to `.eslintrc.json` under `rules` then this will be checked as part of `lint` command
```
"@typescript-eslint/naming-convention": [
      "error",
      {
        "selector": "interface",
        "format": ["PascalCase"],
        "custom": {
          "regex": "^I[A-Z]",
          "match": false
        }
      }
    ]
```